### PR TITLE
feat(flags): Added --stop-on-failure flag

### PR DESCRIPTION
**Pull request by ecktom**

### CODE REVIEW

**Comment by jhedev** [comment]
btw: i would love to see some tests here :)

File: main.go [hunk 1]
```diff
@@ -6,40 +6,68 @@ import (
...
+	exitcode := 0
+	if err := cmd.Run(); err != nil {
+		fmt.Fprintln(os.Stderr, "Command failed: ", err)
 
-		if err := cmd.Run(); err != nil {
-			fmt.Fprintln(os.Stderr, "Command failed: ", err)
-			os.Exit(1)
+		if exitError, ok := err.(*exec.ExitError); ok {
+			ws := exitError.Sys().(syscall.WaitStatus)
+			exitcode = ws.ExitStatus()
```
Comment (philippfranke) [inline main.go]: What about `return ws.ExitStatus()`?

File: main.go [hunk 2]
```diff
@@ -6,40 +6,68 @@ import (
...
+		} else {
```
Comment (philippfranke) [inline main.go]: Remove the else statement and continue with the functions inside

**Comment by ecktom** [comment]
@philippfranke: Improved ;)

**Pull request merged**